### DISCUSSION TITLE
2019/creating-a-bootstrap-based-front-end-for-your-simple-rest-service/

### DISCUSSION THREAD

**Discussion by giscus** [hellt/netdevops.me#17] [Show and tell]
# 2019/creating-a-bootstrap-based-front-end-for-your-simple-rest-service/

Applying DevOps to networks.

https://netdevops.me/2019/creating-a-bootstrap-based-front-end-for-your-simple-rest-service/

**Comment by vsurresh** [comment]
Excellent writeup. I'm a network engineer and manage my own blog and looking into learning the front end. You covered all the basics and explained very well. Keep up the good work.
